polling_stations/apps/dashboard/templates/dashboard/postcode.html

```diff
@@ -0,0 +1,77 @@
+{% extends "base.html" %}
+{% load i18n %}
+{% load pipeline %}
+
+{% block page_title %}{% trans "Postcode:" %} {{ postcode }}{% endblock page_title %}
+
+{% block extra_page_css %}
+    {% stylesheet 'map' %}
+{% endblock extra_page_css %}
+
+{% block extra_javascript %}
+    {% javascript 'map' %}
+{% endblock extra_javascript %}
+
+
+{% block in_page_javascript %}
+    <script type="text/javascript">
+        (function() {
+
+            var typeIcons = {
+            'pollingstation': 'fa-check-square',
+            'residentialaddress': 'fa-home'};
+
+
+            tiles = L.tileLayer('https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png', {
+                attribution: 'Map data ⓒ <a href="http://openstreetmap.org">OpenStreetMap</a> contributors',
+                subdomains: 'abc'
+            });
+
+            map = L.map('area_map', {
+                layers: [tiles]
+            });
+            fetch('{% url "dashboard:postcode-geojson" postcode %}').then((response) => {
+                response.json().then((json) => {
+                    layer = L.geoJSON(json, {
+                          pointToLayer: (feature, latlng) => {
+                              return L.marker(latlng, {icon: L.ExtraMarkers.icon({
+          icon: typeIcons[feature.properties.type] || 'fa-question',
+          markerColor: feature.properties.color,
+          iconColor: 'white',
+          shape: 'circle',
+          prefix: 'fa'
+        })});
```

Comment:
Suggestion: stick  `).bindPopup(feature.properties.address);` on the end of the Marker object. UPRN could be nice too, but is easily cross referenced from the list below.

Reply:
Done, with address, UPRN and link :-)